lib/main.js

```diff
@@ -532,15 +523,15 @@ function GetMML(result) {
 }
 
 //
-//  Creates the speech string and updates the MathML to include it, if needed
+//  Creates dummy speech string and updates the MathML to include it, if needed
 //
 function GetSpeech(result) {
-  if (!data.speakText) return;
-  speech.setupEngine({semantics: true, domain: data.speakRuleset, style: data.speakStyle});
-  result.speakText = speech.toSpeech(result.mml);
+  result.speakText = "Equation";
+  if (data.format !== "MathML") result.speakText = data.math;
   if (!data.mml) return;
   var jax = MathJax.Hub.getAllJax()[0];
-  jax.root.alttext = result.speakText;
+  if (jax.root.alttext) result.speakText = jax.root.alttext;
```

Comment:
Since I'm not sure how the reviews are pushed out in notifications: I just drastically edited my earlier comment.